setup.cfg

```diff
@@ -16,6 +16,7 @@ markers =
 [flake8]
 max-line-length = 88
 exclude = build/*, dist/*, pip_tools.egg-info/*, piptools/_compat/*, .tox/*, .venv/*, .git/*, .eggs/*
+extend-ignore = E203
```

Comment:
Couple you add a comment with a link to explain why we ignore this check?

Reply:
Sure, comment with link added.